api/lib/opentelemetry/instrumentation/registry.rb

```diff
@@ -7,79 +7,79 @@
 module OpenTelemetry
   module Instrumentation
     # The instrumentation Registry contains information about instrumentation
-    # adapters available and facilitates discovery, installation and
+    # instrumentations available and facilitates discovery, installation and
     # configuration. This functionality is primarily useful for SDK
     # implementors.
     class Registry
       def initialize
         @lock = Mutex.new
-        @adapters = []
+        @instrumentations = []
```

Comment:
Should we just call this `@instrumentation`?

Reply:
sure that's fine with me, updated